sycl/test-e2e/KernelCompiler/sycl_cache_pm.cpp

```diff
@@ -25,11 +24,11 @@
 
 // CHECK-WRITTEN-TO-CACHE: [Persistent Cache]: enabled
 // CHECK-WRITTEN-TO-CACHE-NOT: [Persistent Cache]: using cached device binary
-// CHECK-WRITTEN-TO-CACHE: [Persistent Cache]: device binary has been cached
+// CHECK-WRITTEN-TO-CACHE: {{\[Persistent Cache\]: device binary has been cached|\[kernel_compiler Persistent Cache\]: storing device code IR}}
```

Comment:
Intention of this test is to check the PM's caching -- does the test fail without allowing the `[kernel_compiler Persistent Cache] ...` output?